### PR TITLE
Research questions and Sketches

### PR DESCRIPTION
**Pull request by RyKoe**
Created a PR for this to be looked over

### CODE REVIEW

File: proposal.md [hunk 1]
```diff
@@ -18,14 +18,18 @@ Numerical game details include features such as min/max players and min/max/ave
...
 
 The second tab allows the user to explore the most popular boardgames (based on average user rating) filtering on some or all of: categories, mechanics, publisher (using dropdown menus) and published year (using a slider). The user can also select the number of games they want shown using a slider. Features for each of these top selected games will be shown in table format and the average user ratings will also be shown on a bar chart.
 
 An additional tab that may be provided as part of the app (not shown in the sketch below) depending on the timeline includes a two- or three-dimensional plot showing clustering of similar board games likely completed using a dimensionality reducing technique such as Principle Component Analysis. The plot would allow a user to highlight specific categories, mechanics, and publishers.
 
-Include app sketch here
+![dashboard 1](https://github.com/ubco-mds-2020-labs/dashboard-project-group14/tree/main/images/dashboard_sketch_1.PNG)
+
+![dashboard 2](https://github.com/ubco-mds-2020-labs/dashboard-project-group14/tree/main/images/dashboard_sketch_2.PNG)
```
Comment (WraySmith) [inline proposal.md:35]: recommend keeping these as local relative links so they open on any machine and in any branch `./images/<name>`

File: proposal.md [hunk 2]
```diff
@@ -18,14 +18,18 @@ Numerical game details include features such as min/max players and min/max/ave
...
 
 The second tab allows the user to explore the most popular boardgames (based on average user rating) filtering on some or all of: categories, mechanics, publisher (using dropdown menus) and published year (using a slider). The user can also select the number of games they want shown using a slider. Features for each of these top selected games will be shown in table format and the average user ratings will also be shown on a bar chart.
 
 An additional tab that may be provided as part of the app (not shown in the sketch below) depending on the timeline includes a two- or three-dimensional plot showing clustering of similar board games likely completed using a dimensionality reducing technique such as Principle Component Analysis. The plot would allow a user to highlight specific categories, mechanics, and publishers.
 
-Include app sketch here
+![dashboard 1](https://github.com/ubco-mds-2020-labs/dashboard-project-group14/tree/main/images/dashboard_sketch_1.PNG)
```
Comment (WraySmith) [inline proposal.md:33]: images should have labels (below figure): *Figure 1: Board Game Dashboard Sketch (Tab 1)* and *Figure 2: Board Game Dashboard Sketch (Tab 2)*

**Comment by WraySmith** [comment]
Looks good, just a couple comments on the links/labels for the figures.

Also, when opened locally, I got a warning from GitLens about the git history. @mqharris do you get a similar warning - should probably deal with this if it's an issue.

**Comment by mqharris** [comment]
> Looks good, just a couple comments on the links/labels for the figures.
> 
> Also, when opened locally, I got a warning from GitLens about the git history. @mqharris do you get a similar warning - should probably deal with this if it's an issue.

No warnings on my end

File: proposal.md (hunk 1 above)
Comment (RyKoe) [reply proposal.md:35]: Ah yes good idea

**Pull request merged**